EXPLAINER_BLUETOOTH.md

```diff
@@ -177,32 +177,50 @@ method.
 
 Browser implementations may persist Bluetooth serial port access and return them
 to the caller. User agents may remember permission decisions for particular
-devices based on available device properties.Note that a persisted Bluetooth
+devices based on available device properties. Note that a persisted Bluetooth
 serial port may no longer be in communications range. Browser implementations
 may choose to filter out devices that cannot be reached, but this is not
 required. If the browser does not check for device availability while resolving
 `Serial.getPorts()`, `SerialPort.open()` will fail just as it would had that
 Bluetooth device been mapped as a serial port by the OS.
 
-### Events
+### Connection state
 
-Implementation of the
+An implementation of the
 [`onconnect`](https://wicg.github.io/serial/#onconnect-attribute) and
 [`ondisconnect`](https://wicg.github.io/serial/#ondisconnect-attribute) port
-availability callbacks are not required. Implementation is recommended and the
-expectation is that browsers will make a "best effort" to emit these events. If
-a browser is aware of a change in a device's availability it must fire an
-`onconnect`/`ondisconnect` event. Providing an implementation equivalent to a
-wired serial port would necessitate that the host be in a continuous state of
-scanning for Bluetooth devices – as long as at least one remembered Bluetooth
-port is unavailable. This would negatively impact battery life and other system
-resources.
+availability callbacks equivalent to wired serial ports would necessitate that
+the host be in a continuous state of scanning for Bluetooth devices. This would
+negatively impact battery life and other system resources. Instead, events for
+wireless serial ports will be dispatched when the wireless device exposing the
+serial interface becomes connected or disconnected.
+
+The specification will be extended to specify when to dispatch `onconnect` and
+`ondisconnect` events for wireless serial ports. A new concept of a "logically
+connected" serial port will be introduced. A serial port is logically connected
```

Comment:
I think this would be clearer if you removed the sentence in between and linked these paragraphs more closely:

```suggestion
negatively impact battery life and other system resources. Instead, a new concept
of a "logically connected" state for a serial port will be introduced.

The specification will be extended to specify that `onconnect` and `ondisconnect`
events are dispatched for wireless serial ports when this state changes. A serial
port is logically connected
```

Reply:
Done, thanks for the review